src/libraries/System.Diagnostics.Tracing/tests/CustomEventSources/EventSourceTest.cs

```diff
@@ -26,8 +26,9 @@ namespace SdtEventSources
     [EventSource(Guid = "69e2aa3e-083b-5014-cad4-3e511a0b94cf", Name = "EventSourceTest")]
     public sealed class EventSourceTest : EventSource
     {
-        public EventSourceTest(bool useSelfDescribingEvents = false)
-            : base(true)
```

Comment:
looks like you may have been missing on some coverage here since `useSelfDescribingEvents` was being ignored